op-node/p2p/prepared.go

```diff
@@ -68,6 +68,10 @@ func (p *Prepared) PeerScoringParams() *pubsub.PeerScoreParams {
 	return nil
 }
 
+func (p *Prepared) PeerBandScorer() *BandScoreThresholds {
+	return nil
```

Comment:
Does this cause problems in the e2e tests? I don't think we check for the scorer being nil. I think we'd need metrics on in the e2e tests to cause problems here.